### ISSUE TITLE
Error installing lavis manually on windows

### ISSUE THREAD

**Issue by ntrouve-onera** [Tps-F/sd-webui-blip2#20]
Hi, I got an error trying to manually install lavis : 

Building wheels for collected packages: pycocotools
  Building wheel for pycocotools (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for pycocotools (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [24 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-310
      creating build\lib.win-amd64-cpython-310\pycocotools
      copying pycocotools\coco.py -> build\lib.win-amd64-cpython-310\pycocotools
      copying pycocotools\cocoeval.py -> build\lib.win-amd64-cpython-310\pycocotools
      copying pycocotools\mask.py -> build\lib.win-amd64-cpython-310\pycocotools
      copying pycocotools\__init__.py -> build\lib.win-amd64-cpython-310\pycocotools
      running build_ext
      C:\Users\ntrou\AppData\Local\Temp\pip-build-env-kk1efs24\overlay\Lib\site-packages\Cython\Compiler\Main.py:381: FutureWarning: Cython directive 'language_level' not set, using '3str' for now (Py3). This has changed from earlier releases! File: C:\Users\ntrou\AppData\Local\Temp\pip-install-e3akoyw_\pycocotools_9dd8a3ea92f0467e9fc5089539b49339\pycocotools\_mask.pyx
        tree = Parsing.p_module(s, pxd, full_module_name)
      Compiling pycocotools/_mask.pyx because it changed.
      [1/1] Cythonizing pycocotools/_mask.pyx
      building 'pycocotools._mask' extension
      creating build\temp.win-amd64-cpython-310
      creating build\temp.win-amd64-cpython-310\Release
      creating build\temp.win-amd64-cpython-310\Release\common
      creating build\temp.win-amd64-cpython-310\Release\pycocotools
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -IC:\Users\ntrou\AppData\Local\Temp\pip-build-env-kk1efs24\overlay\Lib\site-packages\numpy\core\include -I./common "-IE:\Creation Jeu 2D\IA\stable\stable-diffusion-webui\venv\include" -IC:\Users\ntrou\AppData\Local\Programs\Python\Python310\include -IC:\Users\ntrou\AppData\Local\Programs\Python\Python310\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.27.29110\include" /Tc./common/maskApi.c /Fobuild\temp.win-amd64-cpython-310\Release\./common/maskApi.obj
      maskApi.c
      ./common/maskApi.c(8): fatal error C1083: Impossible d'ouvrir le fichier includeÿ: 'math.h'ÿ: No such file or directory
      error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Tools\\MSVC\\14.27.29110\\bin\\HostX86\\x64\\cl.exe' failed with exit code 2
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pycocotools
Failed to build pycocotools
ERROR: Could not build wheels for pycocotools, which is required to install pyproject.toml-based projects
WARNING: Ignoring invalid distribution -illow (e:\creation jeu 2d\ia\stable\stable-diffusion-webui\venv\lib\site-packages)

Any idea why?

**Comment by Tps-F** [comment]
https://github.com/Tps-F/sd-webui-blip2#install-build-tools
Have you installed the Windows 11 SDK?

**Comment by jndietz** [comment]
Yeah, I'm having the same issue.  I install the build tools as directed but still getting the error.  I'm on Windows 10 still, if that makes any difference.

**Comment by ntrouve-onera** [comment]
For me installing the build tools did solve the error. I did the install process lineary, the build tools install instructions should be moved further up.  Not closing because jndietz seems to have an issue still.

**Comment by Tps-F** [comment]
@jndietz Do you have the Windows 11 SDK installed?
Try the procedure again!
If you can't solve the problem, I'd love for you to send me the actual logs, etc.

**Comment by jndietz** [comment]
I'm not really sure what happened, but I installed the Windows 11 SDK and checked the "Desktop" box under modify and installed it.  But it didn't "work", so I uninstalled it, and re-installed it again and it seemed to work that time.  I am now able to use the extension in automatic1111.
